doc/conf.py

```diff
@@ -31,7 +31,7 @@
 
 import sys
 import os
-# import shlex
+import shlex
```

Comment:
Is there a change?

Reply:
no but I dont think it should be commented-out

Reply:
shlex isn't used. Or is a needed in an other way?